holidays/countries/cocos_islands.py

```
@@ -0,0 +1,262 @@
+        2016: (SEP, 13),
+        2017: (SEP, 1),
+        2019: (AUG, 11),
+        2020: (JUL, 31),
+        2021: (JUL, 20),
+        2022: (JUL, 9),
+        2023: (JUN, 28),
+        2024: (JUN, 17),
+        2025: (JUN, 7),
+        2026: (MAY, 27),
+    }
+     
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)